src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/Commands/Paye/WhenPayeSchemeAdded.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
```

Comment:
We could rename this file to `PayeSchemeAddedCommandHandlerTests` to fit the pattern of existing test filenames.

Reply:
Done